TAP version 13
1..2
not ok 1 - ansible_bad_1.yml
  ---
  message: DEBUG    Logging initialized to level 10\nDEBUG    Options  Namespace(listrules=False, format='rich', quiet=True, parseable=True, parseable_severity=False, progressive=False, rulesdir=[], use_default_rules=True, display_relative_path=True, tags=[], listtags=False, verbosity=2, skip_list=['602', '204', '301', '303', '305', '503'], warn_list=['experimental'], colored=False, exclude_paths=[], config_file='/tmp/lint/.github/linters/.ansible-lint.yml', playbook=['/tmp/lint/.automation/test/ansible/ansible_bad_1.yml'])\nDEBUG    Loading rules from /usr/local/lib/python3.9/site-packages/ansiblelint/rules\nDEBUG    Examining ansible_bad_1.yml of type playbook\nTraceback (most recent call last) \n  File "/usr/local/bin/ansible-lint", line 8, in <module>\n    sys.exit(main())\n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/__main__.py", line 158, in main\n    matches = _get_matches(rules, options)\n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/__main__.py", line 240, in _get_matches\n    matches.extend(runner.run())\n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/runner.py", line 91, in run\n    self.rules.run(file, tags=set(self.tags),\n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/rules/__init__.py", line 236, in run\n    matches.extend(rule.matchtasks(playbookfile, text))\n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/rules/__init__.py", line 88, in matchtasks\n    yaml = append_skipped_rules(yaml, text, file['type'])\n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/skip_utils.py", line 62, in append_skipped_rules\n    yaml_skip = _append_skipped_rules(pyyaml_data, file_text, file_type)\n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/skip_utils.py", line 112, in _append_skipped_rules\n    for ruamel_task, pyyaml_task in zip(ruamel_tasks, pyyaml_tasks) \n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/skip_utils.py", line 160, in _get_tasks_from_blocks\n    for sub_task in get_nested_tasks(task) \n  File "/usr/local/lib/python3.9/site-packages/ansiblelint/skip_utils.py", line 156, in <genexpr>\n    for subtask in task[k]\nTypeError  'NoneType' object is not iterable\n
  ...
ok 2 - ansible_good_1.yml
